e2e/cypress/integration/channel/message_permalink.js

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Please change filename to `message_permalink_spec.js` so that it will be picked by Cypress as one of the test files.